provider/resources.go

```diff
@@ -248,6 +250,10 @@ func Provider() tfbridge.ProviderInfo {
 		},
 	}
 
+	defaults := x.TokensSingleModule("gitlab_", gitLabMod, x.MakeStandardToken(gitLabPkg))
+	err := x.ComputeDefaults(&prov, defaults)
+	contract.AssertNoError(err)
```

Comment:
Apparently we need to use `AssetNoErrorf` now to make the linter happy, haven't seen that before 
```suggestion
	contract.AssertNoErrorf(err)
```

Reply:
race condition: i just pushed that fix, waiting for the tests to rerun